tf2_gnn/layers/nodes_to_graph_representation.py

```diff
@@ -174,9 +186,13 @@ def call(self, inputs: NodesToGraphRepresentationInput, training: bool = False):
                 raise ValueError()
 
         # (2) compute representations for each node/head pair:
-        node_reprs = self._transformation_mlp(
-            inputs.node_embeddings, training=training
+        node_reprs = self._transformation_mlp_activation_fun(
```

Comment:
Adding the missing activation function: We did this somewhere else very recently, how is this related (is it the base class)?

Reply:
Found it, it was in the padded_node_to_graph_representation layer, not in this repo. Not a base class, but very similar.

Reply:
I didn't remember this at all, but you're right: We did this for `PaddedWeightedSumGraphRepresentation` in the other repo. That class essentially a code copy of this (but avoiding the complexity of using sparse operations). So conceptually related, but no formal relationship in the inheritance sense.